### ISSUE TITLE
Invalid attribute "subscribe" at io-package.json

### ISSUE THREAD

**Issue by mcm1957** [iobroker-community-adapters/ioBroker.ical#662]
"subscribe": "ical.INSTANCE.trigger",

configured at io-package.json is no longer supported / valid according to schema.

**Comment by klein0r** [comment]
js-controller 4.x

https://github.com/ioBroker/ioBroker.js-controller/blob/5645debf3394177abeef58917f872a7d8c43e320/schemas/io-package.json#L1042-L1045

js-controller 5.x

https://github.com/ioBroker/ioBroker.js-controller/blob/deb93ac6f9cb5acbd2bdcd1defcd59e6b55fbd6c/schemas/io-package.json#L1295-L1298

Removed in the current master branch? see https://github.com/ioBroker/ioBroker.js-controller/commit/938507b1f67308408f3bc96974468e85dea34dd2

**Comment by mcm1957** [comment]
Yes @Apollon77 and @Foxriver confirmed that "subscribe" has been removed with js-controller 6.

A search at guthub seems to indicate that only iobroker.ical is using this attribute to subscribe  to a state.
Several other adapters use "subsribe":"messagebox" or "subscribe":false. But subsrcibing to messagebox has been replaced by messagebox:true since some time.

**Comment by mcm1957** [comment]
seems to be fixed
